src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,10 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# only set the custom annotation menu labels if we want custom annotations to appear 
+custom_annotation.binary.menu_label =
+custom_annotation.multi.menu_label =
+
+# select the custom annotations by default
+custom_annotation.default = false
```

Comment:
Clarify comment text that this is specific for the OncoPrint view and regards custom driver/passenger annotation. I would also rename the property (custom_annotation.default) to something more descriptive such as `select_custom_driver_passenger_default` or `select_custom_driver_class_default`